routes/defaults.js

```diff
@@ -23,38 +26,55 @@ module.exports = function (log, P, db, error) {
           reply(
             {
               version: version,
-              commit: commitHash
+              commit: commitHash,
+              dbVer: dbVer
             }
           )
         }
 
-        // if we already have the commitHash, send the reply and return
-        if (commitHash) {
-          return sendReply()
-        }
+        // Use async parallel to carry out these various version collection tasks.
```

Comment:
See https://github.com/rkatic/p#api we can use `P.all` or maybe `P.spread` instead of `async`

i.e https://github.com/mozilla/fxa-auth-server/blob/cebd88ffa511f5ad52f6df641cb71b42a79afe46/test/local/email_validity_tests.js#L74